.github/workflows/sync_pytorch_canary.yml

```diff
@@ -9,7 +9,12 @@ on:
 
 jobs:
   sync-pytorch:
+    strategy:
+      matrix:
+        ref: ["master", "viable/strict"]
```

Comment:
Do we also need the nightly branch¿

Reply:
Probably not, this was mainly for the test stats